client/signup/controller.js

```diff
@@ -225,6 +226,31 @@ export default {
 
 		store.set( 'signup-locale', localeFromParams );
 
+		const isOnboardingFlow = flowName === 'onboarding';
+		if ( isOnboardingFlow ) {
+			const stepperOnboardingExperimentAssignment = await loadExperimentAssignment(
+				'calypso_signup_onboarding_flow_stepper'
```

Comment:
The experiment name is called [calypso_signup_onboarding_stepper_flow](https://experiments.a8c.com/experiments/22032-calypso-signup-onboarding-stepper-flow/overview)  not calypso_signup_onboarding_flow_stepper. Please rename